conans/client/build/autotools_environment.py

```diff
@@ -169,8 +128,14 @@ def configure(self, configure_dir=None, args=None, build=None, host=None, target
             pkg_env = {"PKG_CONFIG_PATH": self._conanfile.build_folder} \
                 if "pkg_config" in self._conanfile.generators else {}
 
+        if self._conanfile.package_folder is not None:
+            if not args:
+                args = ["--prefix=%s" % self._conanfile.package_folder.replace("\\", "/")]
+            elif not any(["--prefix=" in arg for arg in args]):
```

Comment:
This doesn't check for `args = ['--prefix', '/foobar']`.

Reply:
But, why is not a single parameter `--prefix=/foobar`?

Reply:
Simply because it is possible. I was successfully using two parameters until yesterday when I upgraded to the latest Conan version and many of my packages broke.